internal/logger/logger.go

```diff
@@ -33,6 +33,8 @@ import (
 // gcsfuse.log in case of GCSFuse.
 const ProgrammeName string = "gcsfuse"
 const GCSFuseInBackgroundMode string = "GCSFUSE_IN_BACKGROUND_MODE"
+const jsonFormat = "json"
```

Comment:
nit: Please add `string` after the names to make it explicit, as it is with the other constants, to keep it consistent.